contract/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+========
+Contract
+========
+
+ * This module recover contracts management with recurring invoicing functions.
```

Comment:
This module forward-port to v9 the contracts management with recurring invoicing functions.